iridauploader/api/api_calls.py

```diff
@@ -699,7 +667,29 @@ def send_sample(self, sample, project_id):
                           "".format(response.status_code, response.text))
             raise self._get_irida_exception(response)
 
-        return json_res
+        return int(json_res['resource']['identifier'])
```

Comment:
I still think you should keep this as the `json_res` as that is what the other `send_*` methods do.

Reply:
Changed.